frontend/src/pages/MessagingQueues/MQDetails/MQTables/MQTables.tsx

```diff
@@ -0,0 +1,165 @@
+import './MQTables.styles.scss';
+
+import { Skeleton, Table, Typography } from 'antd';
+import axios from 'axios';
+import { SOMETHING_WENT_WRONG } from 'constants/api';
+import { QueryParams } from 'constants/query';
+import { useNotifications } from 'hooks/useNotifications';
+import useUrlQuery from 'hooks/useUrlQuery';
+import { isEmpty } from 'lodash-es';
+import {
+	ConsumerLagDetailTitle,
+	ConsumerLagDetailType,
+	convertToTitleCase,
+	RowData,
+	SelectedTimelineQuery,
+} from 'pages/MessagingQueues/MessagingQueuesUtils';
+import { useEffect, useMemo, useState } from 'react';
+import { useMutation } from 'react-query';
+
+import {
+	ConsumerLagPayload,
+	getConsumerLagDetails,
+} from './getConsumerLagDetails';
+
+export function getColumns(data: any): any[] {
```

Comment:
any ?

Reply:
done - under this `https://github.com/SigNoz/signoz/pull/5785/commits/cee05824138f1dc1b027d5658fbbca3419ebf928`